cli/tests/performance_stats.out

```diff
@@ -1,14 +1,14 @@
 [WILDCARD]
-Files: 45
-Nodes: [WILDCARD]
-Identifiers: [WILDCARD]
-Symbols: [WILDCARD]
-Types: [WILDCARD]
-Instantiations: [WILDCARD]
-Parse time: [WILDCARD]
-Bind time: [WILDCARD]
-Check time: [WILDCARD]
-Emit time: [WILDCARD]
-Total TS time: [WILDCARD]
-Compile time: [WILDCARD]
+DEBUG RS - [WILDCARD] - Files: 45
+DEBUG RS - [WILDCARD] - Nodes: [WILDCARD]
+DEBUG RS - [WILDCARD] - Identifiers: [WILDCARD]
+DEBUG RS - [WILDCARD] - Symbols: [WILDCARD]
+DEBUG RS - [WILDCARD] - Types: [WILDCARD]
+DEBUG RS - [WILDCARD] - Instantiations: [WILDCARD]
+DEBUG RS - [WILDCARD] - Parse time: [WILDCARD]
+DEBUG RS - [WILDCARD] - Bind time: [WILDCARD]
+DEBUG RS - [WILDCARD] - Check time: [WILDCARD]
+DEBUG RS - [WILDCARD] - Emit time: [WILDCARD]
+DEBUG RS - [WILDCARD] - Total TS time: [WILDCARD]
+DEBUG RS - [WILDCARD] - Compile time: [WILDCARD]
```

Comment:
What's the reason for this change?

Reply:
If I didn't change it, the tests would fail. Because my patch adds `DEBUG RS -` prefixes to logs with debug level. Previously such prefixes were not there, but they are added in my implementation.